README.md

```diff
@@ -25,7 +25,9 @@ case, that means that when you open a view with the `source.cs` base scope, inst
 
 ## Configuration
 
-Configure OmniSharp by running `Preferences: LSP-OmniSharp Settings` from the Command Palette.
+~~Configure OmniSharp by running `Preferences: LSP-OmniSharp Settings` from the Command Palette.~~
```

Comment:
What does striking it out means? If it should never be used then we should probably remove unnecessary settings from it. But I'm guessing that it's possible to configure it both globally from LSP settings and from a project configuration so both really have its place. Am I right?

Reply:
According official docs [link](https://github.com/OmniSharp/omnisharp-roslyn/wiki/Configuration-Options) and some tests. LSP settings for Omnisharp does not work, Because the configuration not provided by the follow ways.

![image](https://github.com/sublimelsp/LSP-OmniSharp/assets/51814639/7dc206b6-100a-450e-a0ad-2dc9e259c24f)